man/rgblink.5

```diff
@@ -1,53 +1,37 @@
 .\" SPDX-License-Identifier: MIT
 .\"
-.Dd March 28, 2021
+.Dd December 22, 2023
 .Dt RGBLINK 5
 .Os
 .Sh NAME
 .Nm rgblink
 .Nd linker script file format
 .Sh DESCRIPTION
-The linker script is an external file that allows the user to specify the order of sections at link time and in a centralized manner.
+The linker script is a self-contained file that allows specifying attributes for sections at link time, and in a centralized manner.
+There can only be one linker script per invocation of
+.Nm ,
+but it can be split into several files
+.Pq using the Ic INCLUDE No directive .
+.Ss Basic syntax
+A linker script follows a very line-oriented format: each line can contain up to one directive or section name, and up to one comment.
+Whitespace (space and tab characters) is used to separate elements, but is otherwise ignored.
 .Pp
-A linker script consists of a series of bank declarations, each optionally followed by a list of section names (in double quotes) or directives.
-All reserved keywords (bank types and directive names) are case-insensitive; all section names are case-sensitive.
-.Pp
-Any line can contain a comment starting with
+Comments begin with a semicolon
 .Ql \&;
-that ends at the end of the line.
-.Bd -literal -offset indent
-; This line is a comment
-ROMX $F            ; start a bank
-  "Some functions" ; a section name
-  ALIGN 8          ; a directive
-  "Some array"
-
-WRAMX 2            ; start another bank
-  org $d123        ; another directive
-  "Some variables"
-.Ed
-.Pp
-Numbers can be in decimal or hexadecimal format
-.Pq the prefix is Ql $ .
-It is an error if any section name or directive is found before setting a bank.
+character, until the end of the line; they are simply ignored.
 .Pp
-Files can be included by using the
-.Ic INCLUDE
-keyword, followed by a string with the path of the file that has to be included.
+Keywords are composed of letters and digits (but they can't start with a digit); they are all case-insensitive.
 .Pp
-The possible bank types are:
-.Cm ROM0 , ROMX , VRAM , SRAM , WRAM0 , WRAMX , OAM
-and
-.Cm HRAM .
-Unless there is a single bank, which can occur with types
-.Cm ROMX , VRAM , SRAM
-and
-.Cm WRAMX ,
-it is mandatory to specify a bank number after the type.
+Numbers can be written in the usual decimal format, or in binary using the
+.Ql %
+prefix, or in hexadecimal using the
+.Ql $
+prefix (both uppercase and lowercase letters are accepted).
```

Comment:
With the new parser, should we go ahead and support `&octal` and `digit_sep_arators` like rgbasm? (Strictly speaking I don't think anythink in the linker script even needs binary, just dec or hex, but if we keep it because it was already there, may as well support the rest too.)

Reply:
I don't care for octal and thus would rather not add its complexity, and if you can add digit separators without too much hassle, then sure.

Reply:
Okay. I might note explicitly that rgbasm supports octal but rgblink doesn't.